test/closet/config-merge/serverless.yaml

```
@@ -2,8 +2,7 @@ service: my-service
 
 provider:
   name: aws
-  # Maintained to test mixed lambda versions and ensure lower bound of range is valid
-  runtime: nodejs8.10
+  runtime: nodejs12.x
```

Let's keep this so that we ensure the lower part of the support range is covered going forward, even though currently the low part of the range and the high part of the range are the same 😆 .

```suggestion
  # Maintained to test mixed lambda versions and ensure lower bound of range is valid
  runtime: nodejs12.x
```